base/linalg/qr.jl

```diff
@@ -27,7 +27,7 @@ The object has two fields:
     triu(F.factors)` for a `QR` object `F`.
 
   - The subdiagonal part contains the reflectors ``v_i`` stored in a packed format where
-    ``v_i`` is the ``i``th column of the matrix `V = eye(m,n) + tril(F.factors,-1)`.
+    ``v_i`` is the ``i``th column of the matrix `V = I + tril(F.factors, -1)`.
```

Comment:
Sizes `m` and `n` not needed? Same on the changes below.

Reply:
Omitting the explicit `m` and `n` in this documentation seems preferable on this end, particularly if we relax addition with `UniformScaling`s to behave generally as it does with sparse matrices (as discussed on triage).